website/addons/owncloud/settings/defaults.py

```diff
@@ -0,0 +1 @@
+DEFAULT_HOSTS = ['https://localhost/owncloud']
```

Comment:
This should probably be removed from `defaults`, as we wouldn't want it in production. It can stay in `local-dist`, though